css-four-column-layout/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      CSS four column layout challenge
+    </title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h2 class="heading">How it Works</h2>
+      </div>
+      <div class="row">
+        <div class="columns"><img src="images/screen-1.png"> </div>
+        <div class="columns"><img src="images/screen-2.png"></div>
+        <div class="columns"><img src="images/screen-3.png"></div>
+        <div class="columns"><img src="images/screen-4.png"></div>
+      </div>
+      <div class="row">
+        <div class="columns-center">
```

Comment:
there doesn't appear to be a `columns-center` selector in the CSS, not sure what this is doing